src/overlays/actors/ovl_En_Encount2/z_en_encount2.c

```diff
@@ -1,16 +1,30 @@
 #include "z_en_encount2.h"
+#include "overlays/actors/ovl_En_Fire_Rock/z_en_fire_rock.h"
+#include "vt.h"
 
 #define FLAGS 0x00000030
 
 #define THIS ((EnEncount2*)thisx)
 
+typedef enum {
+    /* 0x0 */ INACTIVE,
+    /* 0x1 */ ACTIVE_DEATH_MOUNTAIN,
+    /* 0x2 */ ACTIVE_GANONS_TOWER
+} EnTiteAction;
+
```

Comment:
Id put a `ENCOUNT2_ACTION_` prefix or something on these, and change the enum name
would also use decimal comments personally

Reply:
These aren't actions, they're more like states, I just forgot to change the name of the enum when I copypasted it from EnTite.

I'm going to use this:
```
typedef enum {
    /* 0x0 */ ENCOUNT2_INACTIVE,
    /* 0x1 */ ENCOUNT2_ACTIVE_DEATH_MOUNTAIN,
    /* 0x2 */ ENCOUNT2_ACTIVE_GANONS_TOWER
} Encount2State;
```
Let me know if you disagree but I think it's already clear enough what it is that I don't need to prefix with `STATE_`

Reply:
when I see `ACTORNAME_THING` without any kind of extra description, i automatically assume its a type enum. I do agree that it should be relatively easy to figure that out in context tho.

In the end our standards on this stuff will prob change in documentation phases. it has no chance of conflicting with anything else as it is, so its fine